test/conformance/main_test.go

```diff
@@ -54,3 +64,21 @@ func TestMain(m *testing.M) {
 		return m.Run()
 	}())
 }
+
+func addSourcesInitializers() {
+	apiSrcName := strings.ToLower(fmt.Sprintf("%s",
+		testlib.ApiServerSourceTypeMeta.Kind))
+	pingSrcName := strings.ToLower(fmt.Sprintf("%s",
+		testlib.PingSourceTypeMeta.Kind))
+	sourcesTestRunner.AddComponentSetupClientOption(
+		testlib.ApiServerSourceTypeMeta,
+		setupclientoptions.ApiServerSourceClientSetupOption(apiSrcName,
+			"Reference",
+			recordEventsPodName, roleName, serviceAccountName),
+			)
+	sourcesTestRunner.AddComponentSetupClientOption(
+		testlib.PingSourceTypeMeta,
+		setupclientoptions.PingSourceClientSetupOption(pingSrcName,
+			recordEventsPodName),
+			)
```

Comment:
You need to feed that file to `go fmt`. It's not the only file in which I see formatting "errors", ideally you should configure your editor to format on save.